src/KubernetesClient/LeaderElection/LeaderElector.cs

```diff
@@ -189,11 +189,11 @@ private async Task<bool> TryAcquireOrRenew(CancellationToken cancellationToken)
 
         private async Task AcquireAsync(CancellationToken cancellationToken)
         {
+            var delay = config.RetryPeriod.Milliseconds;
```

Comment:
you can remove `.Milliseconds`, then use `delay *= JitterFactor`

Reply:
https://docs.microsoft.com/en-us/dotnet/api/system.timespan.op_multiply

`*` was introduced after netstandard 2.1
have to revert :(